canonical/juju-dashboard
# WD-4222 - Upgrade Yarn from v1

## Done

- Upgrade to Yarn post v1 (which entered maintenance mode three years ago).

## QA

- [Upgrade](https://yarnpkg.com/getting-started/install) your yarn version to 2+.
- Run `yarn clean`, `yarn install` and then `yarn start`.
- Check that the dashboard runs.

## Details

https://warthogs.atlassian.net/browse/WD-4222
